packages/amplify-category-api/commands/api/gql-compile.js

```diff
@@ -6,8 +6,7 @@ module.exports = {
     try {
       await context.amplify.executeProviderUtils(context, 'awscloudformation', 'compileSchema', { noConfig: true, forceCompile: true });
     } catch (err) {
-      context.print.info(err.stack);
-      context.print.error(err.message);
+      context.print.error(err.toString());
```

Comment:
It would be nice to have an easily identifiable, customer friendly message in every catch before the technical things like stacktrace, here is like:
```context.print.error('There was an error while compiling the schema.');```